plugin.video.twit/addon.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.twit"
+             name="TWiT"
+             version="2.1.8"
+             provider-name="divingmule">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.requests" version="2.22.0"/>
+    <import addon="script.module.feedparser" version="5.2.1"/>
```

Comment:
already py3 compatible?

Reply:
Yes, with a small change to the urllib import but, there are dependency issues with the feedparser and the downloader modules with kodi v19.
Should I make fixes and re-submit to the krypton branch?

Reply:
That's fine. Kodi will have a breaking change in Matrix so you'll need to resubmit to matrix at some point